### PR TITLE
Added raw_mode response for entry

### PR DESCRIPTION
**Pull request by rubydog**
Fixes PY-21

### CODE REVIEW

File: contentful/client.py [hunk 1]
```diff
@@ -219,10 +219,14 @@ def entry(self, entry_id, query=None):
...
             query.update({'sys.id': entry_id})
-            return self._get(
+            response = self._get(
                 self.environment_url('/entries'),
                 query
-            )[0]
+            )
+            if self.raw_mode:
+                return response
+            else:
```
Comment (dlitvakb) [inline contentful/client.py]: No need for this else statement, just do `return response[0]`.

Also please add a blank line between the end of the function call and the returns section.

**Review by dlitvakb — CHANGES_REQUESTED** [review]
Just a minor style change, to follow best practices, and remove one `else` that is not needed.

**Review by dlitvakb — APPROVED** [review]

File: contentful/client.py [hunk 2]
```diff
@@ -219,10 +219,14 @@ def entry(self, entry_id, query=None):
...
-            return self._get(
+            response = self._get(
                 self.environment_url('/entries'),
                 query
-            )[0]
+            )
+
+            if self.raw_mode:
+                return response
+            return response[0]
```
Comment (gilesbrunning) [inline contentful/client.py:229]: Trying to fetch a single entry with `raw_mode=True` throws this error for me:

`TypeError: 'Response' object is not subscriptable`

Comment (rubydog) [reply contentful/client.py:229]: @gilesbrunning I am taking a look at this issue now. Will keep you posted on my progress

Cheers

Comment (rubydog) [reply contentful/client.py:229]: @gilesbrunning I could not reproduce the error. How are you retrieving a single entry? Is there a specific entry you are facing error with? Which version of contentful.py library are you using? If it is not latest can you please update it to latest version and let me know if the problem still persists.

Cheers

Comment (gilesbrunning) [reply contentful/client.py:229]: Hi @rubydog , I was using 1.12.3 but when I upgraded to 1.13.1 the problem is gone so all good. Thanks for checking.

**Pull request merged**